.github/workflows/colima-tests.yml

```diff
@@ -34,16 +34,12 @@ env:
   HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: TRUE
 
-
 permissions:
   contents: read
 
 jobs:
   tests:
     name: colima-${{ matrix.tests }}-no-bind-mounts=${{ matrix.no-bind-mounts }}
-    defaults:
-      run:
-        shell: bash
```

Comment:
I think this exists because the default shell for macOS these days is zsh. Why remove it?

Reply:
Because it's already defined above.